examples/tv-casting-app/linux/main.cpp

```diff
@@ -492,10 +543,43 @@ void ReadServerClustersForNode(NodeId nodeId)
             {
                 ReadServerClusters(binding.remote);
             }
+            else
+            {
+                TargetEndpointInfo * endpointInfo = gTargetVideoPlayerInfo.GetEndpoint(binding.remote);
+                if (endpointInfo != nullptr && endpointInfo->IsInitialized())
+                {
+                    endpointInfo->PrintInfo();
+                }
+            }
         }
     }
 }
 
+CHIP_ERROR ContentLauncherLaunchURL(const char * contentUrl, const char * contentDisplayStr)
+{
+    OperationalDeviceProxy * operationalDeviceProxy = gTargetVideoPlayerInfo.GetOperationalDeviceProxy();
+    if (operationalDeviceProxy == nullptr)
+    {
+        ChipLogError(AppServer, "Failed in getting an instance of OperationalDeviceProxy");
+        return CHIP_ERROR_PEER_NODE_NOT_FOUND;
+    }
+
+    ContentLauncherCluster cluster;
+    CHIP_ERROR err = cluster.Associate(operationalDeviceProxy, kTvEndpoint);
+    if (err != CHIP_NO_ERROR)
+    {
+        ChipLogError(AppServer, "Associate() failed: %" CHIP_ERROR_FORMAT, err.Format());
+        return err;
+    }
+    LaunchURL::Type request;
+    request.contentURL          = chip::CharSpan::fromCharString(contentUrl);
+    request.displayString       = Optional<CharSpan>(chip::CharSpan::fromCharString(contentDisplayStr));
+    request.brandingInformation = Optional<chip::app::Clusters::ContentLauncher::Structs::BrandingInformation::Type>(
+            chip::app::Clusters::ContentLauncher::Structs::BrandingInformation::Type());
```

Comment:
```suggestion
    request.brandingInformation = MakeOptional(
            chip::app::Clusters::ContentLauncher::Structs::BrandingInformation::Type());
```
Or could just not set it at all, since "optional without value" is the default anyway....

Reply:
done